api-reference/10 UI Components/GridBase/1 Configuration/selection/allowSelectAll.md

```diff
@@ -5,9 +5,26 @@ default: true
 ---
 ---
 ##### shortDescription
-Specifies whether a user can select all rows at once.
+Specifies whether a user can select all (default) or current page rows at once.
```

Comment:
Instead of "specifies whether a user can or cannot select", this description reads as "specifies whether a user can select all or current page rows".

```suggestion
Allows users to simultaneously select all or current page rows (depending on the [selectAllMode]({basewidgetpath}/Configuration/selection/#selectAllMode)).
```